providers/implementations/kdfs/tls1_prf.c

```diff
@@ -181,9 +181,6 @@ static int kdf_tls1_prf_set_ctx_params(void *vctx, const OSSL_PARAM params[])
     }
     /* The seed fields concatenate, so process them all */
     if ((p = OSSL_PARAM_locate_const(params, OSSL_KDF_PARAM_SEED)) != NULL) {
-        OPENSSL_cleanse(ctx->seed, ctx->seedlen);
```

Comment:
I've doubts here but don't see a better way.
HKDF does the same magic with its salt and that hasn't been fixed the same way.

Reply:
The HKDF situation is different. Here we are concatenating multiple seed values together - which may come from multiple ctrl calls. That isn't the case in HKDF.

Reply:
Originally it was the same for HKDF.  I thought I fixed the multiple ctrl call issue and build param arrays containing all of the values that were passed through in one go.